<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStandardFeedDetails</class>
 <widget class="QDialog" name="FormStandardFeedDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_lblParentCategory">
       <property name="text">
        <string>Parent category</string>
       </property>
       <property name="buddy">
        <cstring>m_cmbParentCategory</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_cmbParentCategory">
       <property name="toolTip">
        <string>Select parent item for your feed.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_cmbType">
       <property name="toolTip">
        <string>Select type of the standard feed.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_lblTitle">
       <property name="text">
        <string>Title</string>
       </property>
       <property name="buddy">
        <cstring>m_txtTitle</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="LineEditWithStatus" name="m_txtTitle" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_lblDescription">
       <property name="text">
        <string>Description</string>
       </property>
       <property name="buddy">
        <cstring>m_txtDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="LineEditWithStatus" name="m_txtDescription" native="true"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="LineEditWithStatus" name="m_txtUrl" native="true"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Encoding</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="m_cmbEncoding">
       <property name="toolTip">
        <string>Select encoding of the standard feed. If you are unsure about the encoding, then select &quot;UTF-8&quot; encoding.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="m_lblIcon">
       <property name="text">
        <string>Icon</string>
       </property>
       <property name="buddy">
        <cstring>m_btnIcon</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QToolButton" name="m_btnIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select icon for your feed.</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QGroupBox" name="m_gbAuthentication">
       <property name="toolTip">
        <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
       </property>
       <property name="title">
        <string>Requires authentication</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Username</string>
          </property>
          <property name="buddy">
           <cstring>m_txtUsername</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="LineEditWithStatus" name="m_txtUsername" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>m_txtPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="LineEditWithStatus" name="m_txtPassword" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormStandardFeedDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
